sql/create.go

```diff
@@ -42,15 +42,29 @@ func (p *planner) CreateDatabase(n *parser.CreateDatabase) (planNode, error) {
 		}
 		return nil, fmt.Errorf("database \"%s\" already exists", n.Name)
 	}
+
 	ir, err := p.db.Inc(keys.DescIDGenerator, 1)
 	if err != nil {
 		return nil, err
 	}
-	nsID := uint32(ir.ValueInt() - 1)
+
+	desc := makeDatabaseDesc(n)
+	desc.ID = uint32(ir.ValueInt() - 1)
+
+	// TODO(pmattis): The error currently returned below is likely going to be
```

Comment:
This will be the CPut error, right? So it'll be difficult to interpret in the context of SQL but not otherwise obscure, right?